packages/cli/src/nonInteractiveCli.ts

```diff
@@ -79,16 +88,50 @@ export async function runNonInteractive(
             id: toolCallRequest.callId,
           };
           functionCalls.push(fc);
+        } else if (event.type === GeminiEventType.Finished) {
+          chatRecordingService.recordMessageTokens({
```

Comment:
Should this recordMessageTokens just take the usageMetadata actually and pull out all these values?